### PR TITLE
Feature/403

### PR DESCRIPTION
**Pull request by RomanGalochkin**
#403

### CODE REVIEW

File: packages/server/src/models/hooks.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const events = require('../async/events');
+const emitter = require('../async/emitter');
+
+const { MEASURABLE_STATE } = require('./enums/measurable-state');
+
+function addHooks(db) {
+  db.sequelize.addHook('afterUpdate', (instance) => {
```
Comment (OAGr) [inline packages/server/src/models/hooks.js]: why are these here instead of in files related to the model, like models/measurable?

**Comment by OAGr** [comment]
There were a whole lot of small changes made for this PR. Can you briefly describe the main reasons for these?

File: packages/server/src/models/hooks.js (hunk 1 above)
Comment (RomanGalochkin) [reply packages/server/src/models/hooks.js]: The purpose is to encupsulate this part of application. Here "Models" just drivers to DB nothing more.

**Comment by RomanGalochkin** [comment]
Yes, sure. Previously when we started to implement "models drivers"/"models"/"datas layers" template I have made an error in interfaces definitions, so to repair this situation I need to rename all legacy methods of "data layer" into to something "nameOfMethod2" to use parents good written methods "getOne" and another. This will keep lines of code on the next stages of application live.
